net/libpsl/Portfile

```diff
@@ -49,7 +49,7 @@ checksums           ${main_distfile} \
 # See e.g. https://trac.macports.org/ticket/60419
 
 # DO NOT change this unless you have understood and acted on the above comment!
-set py_ver          3.10
+set py_ver          3.12
```

Comment:
I am pretty sure you did not read or understood the comment above here and/or verified that this didn't cause a problem. There are also many other, unrelated commits in this PR - so closing